packages/sdk-ark/source/client.service.ts

```diff
@@ -216,6 +216,15 @@ export class ClientService extends Services.AbstractClientService {
 
 			result.searchParams = dotify({ ...result.searchParams, ...result.body });
 			result.body = null;
+		} else {
+			// @ts-ignore
+			const identifiers: WalletIdentifier[] | undefined = body.identifiers as WalletIdentifier[];
+			if (identifiers) {
+				result.body.address = identifiers.map(({ value }) => value);
```

Comment:
```suggestion
				result.body.addresses = identifiers.map(({ value }) => value);
```

If this applies to v2.

Reply:
That's how it is expected (I guess that's the explorer API).

Reply:
Oh wait

Reply:
Checking...